TextFormatter.java

```diff
@@ -1,26 +1,85 @@
+import java.util.ArrayList;
+import java.util.StringTokenizer;
+
 class TextFormatter {
 
-  private static final String text = "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy " +
-          "eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et " +
-          "accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem " +
-          "ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod " +
-          "tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et " +
-          "justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est " +
-          "Lorem ipsum dolor sit amet.";
-
-  public static void main(String[] args) {
-    TextFormatter formatter = new TextFormatter(30);
-    formatter.print(text);
-  }
+	static int maxLineLength_;
+	static int letterCounter = 0;
+	static String formattedText = "";
+	static ArrayList<String> words;
 
-  // Konstruktor
-  public TextFormatter(int maxLineLength) {
-    // ...
-  }
+	private static final String text = "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy "
+			+ "eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et "
+			+ "accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem "
+			+ "ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod "
+			+ "tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et "
+			+ "justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est "
+			+ "Lorem ipsum dolor sit amet.";
 
-  // Ausgabe
-  public void print(String aText) {
-    System.out.println("Hier sollte der Text mit passendem Umbruch erscheinen.");
-  }
+	public static void main(String[] args) {
+		TextFormatter formatter = new TextFormatter(30);
+		formatter.print(text);
+	}
+
+	// Konstruktor
+	public TextFormatter(int maxLineLength) {
+		maxLineLength_ = maxLineLength;
```

Comment:
Die Verwendung des Unterstrichs am Ende finde ich unter Clean Code Aspekten nicht optimal. Ggf. kann man mit this.maxLineLength gezielt die Instanzvariable ansprechen, die sonst durch den gleichnamigen Parameter verdeckt ist.